### PR TITLE
Configuration file added for debugging Next.js in VSCode and removed .vscode from gitignore for the same

### LINKED ISSUE #7438: [OTHER] Support for debugging LinkFree
### What would you like to share?

During my analysis of the LinkFree issue [here](https://github.com/EddieHubCommunity/LinkFree/issues/7065), I noticed that we currently lack any configuration for debugging purposes.

Although debugging locally in VSCode can be a bit challenging, it is still achievable. Unfortunately, I couldn't provide a solution for this matter due to personal laptop issues, as my laptop is broken, and my office laptop has several restrictions in place.

### My Analysis:
During my analysis, I discovered that this issue can be resolved by adding a configuration to the project. Please refer to the screenshot below:
image

For more detailed information, you can visit the following link: [Debugging in Next.js](https://nextjs.org/docs/pages/building-your-application/configuring/debugging)

### Proposed Actions:
1. I kindly request someone to try implementing this solution and, if everything works properly, push it to the repository. Also, consider adding it as a candidate for the `.gitignore` file.
2. It would be beneficial to create documentation that provides a comprehensive guide on debugging for LinkFree.

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by yashpandey002**
## Changes proposed

- Created a new folder .vscode in the root directory.
  - Created a new file launch.json in the .vscode folder.
- Added the configuration code for debugging Next.js in VSCode, from [here](https://nextjs.org/docs/pages/building-your-application/configuring/debugging).
- Removed .vscode from gitignore for making it trackable.

## Check List (Check all the applicable boxes) 

- [X] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [X] This PR does not contain plagiarized content.
- [X] The title of my pull request is a short description of the requested changes.

## Screenshots

image

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/7694"></a>

### CODE REVIEW

**Review by amandamartin-dev — CHANGES_REQUESTED** [review]
Hello! I think this update should have a docs update as well so that folks know how to use it.  Or a little intro that this is configured and a link to external docs on how to use it. wdyt?

**Comment by yashpandey002** [comment]
> Hello! I think this update should have a docs update as well so that folks know how to use it. Or a little intro that this is configured and a link to external docs on how to use it. wdyt?

Hey Amanda, Yes I totally agree with that. In fact, I also have the same thought in mind but I get confused that if I should make the changes to documents in this PR only or make another PR. I never worked on creating documents so I will need some help but will love to work on that. Can you guide me on how should I start?

File: .gitignore [hunk 1]
```diff
@@ -35,5 +35,4 @@ test-results/
 # storybook
 /storybook-static
 
-# vscode
-.vscode
```
Comment (eddiejaoude) [inline .gitignore:39]: This should still be ignored, otherwise we will get many more files added every time someone does a commit

**Comment by eddiejaoude** [comment]
> > Hello! I think this update should have a docs update as well so that folks know how to use it. Or a little intro that this is configured and a link to external docs on how to use it. wdyt?
> 
> Hey Amanda, Yes I totally agree with that. In fact, I also have the same thought in mind but I get confused that if I should make the changes to documents in this PR only or make another PR. I never worked on creating documents so I will need some help but will love to work on that. Can you guide me on how should I start?

Yes ideally the PR should contain any relevant changes, Ravi has a great base for the docs https://github.com/EddieHubCommunity/LinkFree/issues/7438#issuecomment-1582076163

File: .gitignore (hunk 1 above)
Comment (yashpandey002) [reply .gitignore:39]: Okay I got it, so I should only update docs and add something like "How to debug LinkFree's Next.js code on VSCode"?

Comment (eddiejaoude) [reply .gitignore:39]: You can add a specific config file but still ignore the `.vscode` directory because we don't want all the generated files

**Comment by SaraJaoude** [comment]
Due to lack of activity I will be closing this PR.

**Pull request closed without merging**